.php_cs

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once './vendor/autoload.php';
+require __DIR__.'/vendor/sllh/php-cs-fixer-styleci-bridge/autoload.php';
```

Comment:
On puli, the conflict is still here:

```bash
sullivan@sweetnexy:~/projects/fork/puli/cli(styleci-bridge)$ php-cs-fixer fix -v
Fixer "ordered_use" does not exist, skipping.
Fixer "empty_return" does not exist, skipping.
Loaded config from "/home/sullivan/projects/fork/puli/cli/.php_cs".
Using cache file ".php_cs.cache".
PHP Fatal error:  Call to undefined method PhpCsFixer\FixerFileProcessedEvent::setDispatcher() in /home/sullivan/.composer/vendor/symfony/event-dispatcher/EventDispatcher.php on line 42
```

But this happen only with a composer globally installed php-cs-fixer. The `.phar` way works great.

Not sure if this can be easily resolvable.